docs/creating-a-codelist-from-a-csv-file.md

```diff
@@ -9,11 +9,18 @@ and choose a file to upload from your hard drive.
 
         create-a-codelist-1.png
 
-Note that the CSV file should not have a header,
-and its first column must contain valid codes in the chosen coding system.
-(This limitation will be lifted.)
+The final codelist must be stored in CSV format to be imported 
+into [OpenCodelists](https://www.opencodelists.org). The codes must be stored in exactly one column.
+There is currently a soft requirement that the first column must contain codes in the chosen coding system, preferably named according to the CSV column names provided in the table above (this limitation will be lifted). The second column is typically a text description of the code. Before uploading a codelist CSV under your OpenSAFELY codelist profile, you should remove the header row as these will be automatically added in a standardised manner.
```

Comment:
should we encourage/enforce there being a description column? Or are descriptions fetched from our data?

Reply:
I've left this as is for now. Not exactly sure if we fetch descriptions